### PR TITLE
Add sushytools scale up in dev-env

### PR DESCRIPTION
**Pull request by mboukhalfa**

### CODE REVIEW

File: Support/sushytools/scale-to-2-instances.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Sushy-tools scale instances in Dev-env
+
+Dev-env supports redfish BMC using sushy-tools that runs in a container building the image from [https://github.com/metal3-io/ironic-image/tree/main/resources/sushy-tools](https://github.com/metal3-io/ironic-image/tree/main/resources/sushy-tools). To select redfish virtual media in the dev-env `export BMC_DRIVER="redfish-virtualmedia"`.
+
+sushy-tools is backed with libvirt, it listens to redfish requests on `localhost:8000` by default. An instance of sushy-tools shows all the libvirt domains that means multiple instances will be able to see same domains.
+eg: `curl http://localhost:8000/redfish/v1/Systems/`S
```
Comment (tuminoid) [inline Support/sushytools/scale-to-2-instances.md:6]: Extra-S at end of line

**Review by tuminoid — CHANGES_REQUESTED** [review]
some typos

File: Support/sushytools/scale-to-2-instances.md (hunk 1 above)
Comment (mboukhalfa) [reply Support/sushytools/scale-to-2-instances.md]: fixed

File: Support/sushytools/scale-to-2-instances.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+To scale up the sushy-tools instances it suffices to run multiple instaces of the container and configure a different port in the sushy-tools configfile  for each instance. BMO use the bmh spec  to grep the sushy-tools instance ip/port to use.
```
[the remark this answers is not in the document]
Comment (mboukhalfa) [reply Support/sushytools/scale-to-2-instances.md]: fixed

File: Support/sushytools/scale-to-2-instances.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Scaling sushy-tools instances in Dev-env
+
+Dev-env supports redfish BMC using sushy-tools running in a container and building its the image from [https://github.com/metal3-io/ironic-image/tree/main/resources/sushy-tools](https://github.com/metal3-io/ironic-image/tree/main/resources/sushy-tools). To select redfish virtual media in the dev-env export the BMC driver environment variable `export BMC_DRIVER="redfish-virtualmedia"`.
```
Comment (lentzi90) [inline Support/sushytools/scale-to-2-instances.md:3]: nit
```suggestion
Dev-env supports redfish BMC using sushy-tools running in a container and building the image from [https://github.com/metal3-io/ironic-image/tree/main/resources/sushy-tools](https://github.com/metal3-io/ironic-image/tree/main/resources/sushy-tools). To select redfish virtual media in the dev-env export the BMC driver environment variable `export BMC_DRIVER="redfish-virtualmedia"`.
```

**Review by lentzi90 — DISMISSED** [review]

**Review by lentzi90 — DISMISSED** [review]

File: Support/sushytools/scale-to-2-instances.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Scaling sushy-tools instances in Dev-env
+
+Dev-env supports redfish BMC using sushy-tools running in a container and building the image from [https://github.com/metal3-io/ironic-image/tree/main/resources/sushy-tools](https://github.com/metal3-io/ironic-image/tree/main/resources/sushy-tools). To select redfish virtual media in the dev-env export the BMC driver environment variable `export BMC_DRIVER="redfish-virtualmedia"`.
+
+Sushy-tools is backed with libvirt, and by default it listens to redfish requests on the port `8000`. An instance of sushy-tools sees all the libvirt domains that means multiple instances will be able to see the same domains.
```
Comment (smoshiur1237) [inline Support/sushytools/scale-to-2-instances.md:5]:
```suggestion
Sushy-tools is backed by libvirt and it listens to redfish requests on the port `8000` by default. An instance of sushy-tools can see all the libvirt domains and that means multiple instances will be able to see the same domains.
```

File: Support/sushytools/scale-to-2-instances.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+eg: `curl http://localhost:8000/redfish/v1/Systems/`
+
+To scale up the sushy-tools instances it suffices to run multiple instances of the container and configure a different port for each instance in the sushy-tools configfile.
```
Comment (smoshiur1237) [inline Support/sushytools/scale-to-2-instances.md:8]:
```suggestion
To scale up the sushy-tools instances, it suffices to run multiple instances of the container and configure a different port for each instance in the sushy-tools configfile.
```

**Review by smoshiur1237 — CHANGES_REQUESTED** [review]
Nit

**Review by smoshiur1237 — COMMENTED** [review]
LGTM

File: Support/sushytools/scale-to-2-instances.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# Scaling sushy-tools instances in Dev-env
+
+Dev-env supports redfish BMC using sushy-tools running in a container and building the image from [https://github.com/metal3-io/ironic-image/tree/main/resources/sushy-tools](https://github.com/metal3-io/ironic-image/tree/main/resources/sushy-tools). To select redfish virtual media in the dev-env export the BMC driver environment variable `export BMC_DRIVER="redfish-virtualmedia"`.
+
+Sushy-tools is backed by libvirt and it listens to redfish requests on the port `8000` by default. An instance of sushy-tools can see all the libvirt domains and that means multiple instances will be able to see the same domains.
+eg: `curl http://localhost:8000/redfish/v1/Systems/`
```
Comment (furkatgofurov7) [inline Support/sushytools/scale-to-2-instances.md:6]:
```suggestion
e.g.: `curl http://localhost:8000/redfish/v1/Systems/`
```

File: Support/sushytools/scale-to-2-instances.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+To scale up the sushy-tools instances, it suffices to run multiple instances of the container and configure a different port for each instance in the sushy-tools configfile.
```
Comment (furkatgofurov7) [inline Support/sushytools/scale-to-2-instances.md:8]:
```suggestion
To scale up the sushy-tools instances, it is sufficient to run multiple instances of the container and configure a different port for each instance in the sushy-tools configfile.
```

Comment (mboukhalfa) [reply Support/sushytools/scale-to-2-instances.md]: fixed

File: Support/sushytools/scale-to-2-instances.md (hunk 6 above)
Comment (mboukhalfa) [reply Support/sushytools/scale-to-2-instances.md]: fixed

**Review by smoshiur1237 — COMMENTED** [review]
LGTM

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Review by furkatgofurov7 — APPROVED** [review]

**Review by smoshiur1237 — APPROVED** [review]

**Review by tuminoid — APPROVED** [review]

**Pull request merged**